pkg/sentry/sentry.go

```diff
@@ -104,45 +108,43 @@ func (s *sentry) Start(ctx context.Context) error {
 		return fmt.Errorf("error creating security: %s", err)
 	}
 
-	val, err := s.validator(ctx)
+	val, err := s.getValidator(ctx)
 	if err != nil {
 		return err
 	}
 
-	validatorErr := make(chan error, 1)
+	errCh := make(chan error, 3)
```

Comment:
Without this change on how errors are returned, I couldn't get errors from `val.Start()` to make the app exit. It would just hang until SIGTERM.